### PR TITLE
improve deploy list output

### LINKED ISSUE #344: [PROPOSAL] List deploys in reversed order
Currently, the last revision is shown at bottom of the table.
I think in most cases, we only need to know which was the last one, and it would be more easy to see if it is displayed first. Example:

```
+----------+---------+-----+------------------------------------------+
| REVISION | CURRENT | AGE |               DESCRIPTION                |
+----------+---------+-----+------------------------------------------+
| 22       | true    | 4d  | 48f4553 Bump version: 1.118.0 → 1.119.0  |
+----------+---------+-----+------------------------------------------+
| 21       | false   | 5d  | v1.118.0                                 |
+----------+---------+-----+------------------------------------------+
| 20       | false   | 10d | 5290ab57 Bump version: 1.116.0 → 1.117.0 |
+----------+---------+-----+------------------------------------------+
| 19       | false   | 12d | 226868ea Bump version: 1.115.1 → 1.116.0 |
+----------+---------+-----+------------------------------------------+
| 18       | false   | 12d | c00332b Bump version: 1.115.0 → 1.115.1  |
+----------+---------+-----+------------------------------------------+
| 17       | false   | 19d | c2d00239 Bump version: 1.114.2 → 1.115.0 |
+----------+---------+-----+------------------------------------------+
```

### LINKED ISSUE #345: [PROPOSAL] Show current revision together with revision number
Currently, there is a column `CURRENT` with `true/false` values, which only one will be `true`.

I think would be nicer and easier to find which revision is the current if its displayed together with revision number.

Today:

```
+----------+---------+-----+------------------------------------------+
| REVISION | CURRENT | AGE |               DESCRIPTION                |
+----------+---------+-----+------------------------------------------+
| 17       | false   | 19d | c2d00239 Bump version: 1.114.2 → 1.115.0 |
+----------+---------+-----+------------------------------------------+
| 18       | false   | 12d | c00332b Bump version: 1.115.0 → 1.115.1  |
+----------+---------+-----+------------------------------------------+
| 19       | true    | 12d | 226868ea Bump version: 1.115.1 → 1.116.0 |
+----------+---------+-----+------------------------------------------+
| 20       | false   | 10d | 5290ab57 Bump version: 1.116.0 → 1.117.0 |
+----------+---------+-----+------------------------------------------+
| 21       | false   | 5d  | v1.118.0                                 |
+----------+---------+-----+------------------------------------------+
| 22       | false   | 4d  | 48f4553 Bump version: 1.118.0 → 1.119.0  |
+----------+---------+-----+------------------------------------------+
```

```
+--------------+-----+------------------------------------------+
| REVISION     | AGE |               DESCRIPTION                |
+--------------+-----+------------------------------------------+
| 17           | 19d | c2d00239 Bump version: 1.114.2 → 1.115.0 |
+--------------+-----+------------------------------------------+
| 18           | 12d | c00332b Bump version: 1.115.0 → 1.115.1  |
+--------------+-----+------------------------------------------+
| 19 (current) | 12d | 226868ea Bump version: 1.115.1 → 1.116.0 |
+--------------+-----+------------------------------------------+
| 20           | 10d | 5290ab57 Bump version: 1.116.0 → 1.117.0 |
+--------------+-----+------------------------------------------+
| 21           | 5d  | v1.118.0                                 |
+--------------+-----+------------------------------------------+
| 22           | 4d  | 48f4553 Bump version: 1.118.0 → 1.119.0  |
+--------------+-----+------------------------------------------+
```

what do you think?

### PR DESCRIPTION
**Pull request by yagonobre**
fixes #344 
fixes #345

---
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/355)

### CODE REVIEW

File: pkg/client/cmd/deploy.go [hunk 1]
```diff
@@ -335,17 +335,19 @@ func deployList(cmd *cobra.Command, args []string) {
...
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"REVISION", "CURRENT", "AGE", "DESCRIPTION"})
+	table.SetHeader([]string{"REVISION", "AGE", "DESCRIPTION"})
 	table.SetRowLine(true)
 	table.SetAlignment(tablewriter.ALIGN_LEFT)
 	table.SetRowSeparator("-")
 	table.SetAutoWrapText(false)
 
-	sort.Sort(deploy.ByRevision(resp.Deploys))
+	sort.Sort(sort.Reverse(deploy.ByRevision(resp.Deploys)))
```
Comment (drgarcia1986) [inline pkg/client/cmd/deploy.go]: why both `Sort` and `Reverse`?

Comment (aguerra) [reply pkg/client/cmd/deploy.go]: I guess it's easier to change `deploy.ByRevision` (and all the logic stays on the server side).

Comment (yagonobre) [reply pkg/client/cmd/deploy.go]: The `Sort` only support sort in ascending order, the `Reverse` return the reverse order for data.

Comment (drgarcia1986) [reply pkg/client/cmd/deploy.go]: gotcha:
```go
type reverse struct {
	// This embedded Interface permits Reverse to use the methods of
	// another Interface implementation.
	Interface
}
```
`sort/sort.go` in the stdlib...
sorry for the noise.

**Comment by drgarcia1986** [comment]
@yagonobre  Could you add an entry on changelog?

**Comment by yagonobre** [comment]
@drgarcia1986 done

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Next Version] - Release Date
+### Changed
+- deploy list now print revision in reverse orde and remove current column
```
Comment (drgarcia1986) [inline CHANGELOG.md]: 💅 add a line break 💅

Comment (aguerra) [reply CHANGELOG.md]: `reverse order`

**Comment by yagonobre** [comment]
@aguerra @drgarcia1986 rebased

**Pull request merged**